pkg/reconciler/namespace/namespace.go

```diff
@@ -174,6 +185,14 @@ func (r *Reconciler) reconcileServiceAccountAndRoleBindings(ctx context.Context,
 		return fmt.Errorf("track role binding '%s': %v", rb.Name, err)
 	}
 
+	if sa.Name == resources.IngressServiceAccountName || sa.Name == resources.FilterServiceAccountName {
```

Comment:
Perhaps we should first check if the target secret exists instead of always trying to copy it over? That's how the other reconcile loops do it, check and if it's good, don't always create one.